### PR TITLE
bugfix: CollapsiblePanel - caret vertical align

### PR DESCRIPTION
**Pull request by charlesnguyen**
**Please check if the PR fulfills these requirements**

- [x] The commit(s) message(s) follows our [guidelines](https://github.com/talend/tools/blob/master/tools-root-github/CONTRIBUTING.md#commit-message-format)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?**

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [ ] Other... Please describe:

**What is the current behavior?** (You can also link to an open issue here)

**What is the new behavior?**
the caret is vertical aligned on firefox and IE

**Does this PR introduce a breaking change?**

- [ ] Yes
- [x] No

If this PR contains a breaking change, please describe the impact and migration path for existing applications: ...

**Other information**:

### CODE REVIEW

File: src/CollapsiblePanel/CollapsiblePanel.scss [hunk 1]
```diff
@@ -100,6 +103,10 @@ $tc-collapsible-panel-padding-larger: $padding-larger !default;
 			&[aria-expanded="true"] .tc-collapsible-panel-caret-down {
 				transform: rotate(180deg);
 			}
+
+			& > * {
```
Comment (frassinier) [inline src/CollapsiblePanel/CollapsiblePanel.scss]: `&` is useless here

File: src/CollapsiblePanel/CollapsiblePanel.scss [hunk 2]
```diff
@@ -100,6 +103,10 @@ $tc-collapsible-panel-padding-larger: $padding-larger !default;
...
+				margin: auto 15px;
```
Comment (frassinier) [inline src/CollapsiblePanel/CollapsiblePanel.scss]: `15px` is not a known variable value?

File: src/CollapsiblePanel/CollapsiblePanel.scss (hunk 1 above)
Comment (charlesnguyen) [inline src/CollapsiblePanel/CollapsiblePanel.scss]: oki :)

File: src/CollapsiblePanel/CollapsiblePanel.scss (hunk 2 above)
Comment (charlesnguyen) [inline src/CollapsiblePanel/CollapsiblePanel.scss]: No. I just want to override the bootstrap style (margin: 0 15px) of Panel

**Pull request merged**